app/controllers/v1/padlock/registration_controller.rb

```diff
@@ -0,0 +1,19 @@
+module V1
+  module Padlock
+    class RegistrationController < V1::Padlock::PadlockController
+      def create
+        @user = V1::Padlock::Registrations::Create.call(user_params)
+
+        return render json: @user.errors, status: :unprocessable_entity unless @user.persisted?
```

Comment:
`if @user.new_record?` => same

Reply:
`unless persisted` seems to better convey my intention here, namely - I wanna render errors unless the user was saved

Reply:
Ok, just noticed that => `persisted? == !(new_record? || destroyed?)`